app/res/values-de/strings.xml

```diff
@@ -252,8 +252,8 @@
     <string name="code">Code</string>
     <string name="following_user">Folgen…</string>
     <string name="unfollowing_user">Entfolgen…</string>
-    <string name="starring_repository">Hervorheben…</string>
-    <string name="unstarring_repository">Hervorhebung entfernen…</string>
+    <string name="starring_repository">Favorisiere Repository…</string>
```

Comment:
The phrases for starring gists and repositories should match.

When ```starring_gist``` is translated as "Füge Gist als Favorit hinzu" ```starring_repository``` should be "Füge Repository als Favorit hinzu". Or ```starring_gist``` should be "Favorisiere Gist"

In my opinion the long forms "Füge XX als Favorit hinzu" are better